core/templates/pages/story-viewer-page/story-viewer-page.directive.html

```diff
@@ -18,43 +18,44 @@
               <div ng-style="($index > 0 && !$ctrl.storyPlaythroughObject.getStoryNodes()[$index-1].isCompleted() && {'opacity': '0.5', 'display': 'flex'}) || { 'display': 'flex' }">
                 <svg ng-if="!$last"
                      width="130px"
-                     height="300px"
-                     ng-style="height: 300px;"
+                     height="360px"
                      class="oppia-hide-lesson-track"
                      xmlns="http://www.w3.org/1600/svg">
                   <path fill="none"
                         ng-attr-stroke="{{node.isCompleted() && '#296E5F' || 'rgb(204, 204, 204)' }}"
                         stroke-dasharray="30,16"
                         stroke-linecap="round"
-                        stroke-width="10"
-                        d="M70 30 L70 300"/>
+                        stroke-width="8"
+                        d="M71 30 L71 360">
+                  </path>
                 </svg>
-                <div ng-if="$last" style="width: 150px;" class="oppia-hide-lesson-track" />
+                <div ng-if="$last" style="width: 130px;" class="oppia-hide-lesson-track">
```

Comment:
Can you move this inline style into the CSS class?

Reply:
Done.